app/modules/key_management/secret_manager.py

```
@@ -21,13 +23,38 @@
             client = secretmanager.SecretManagerServiceClient()
-            project_id = os.environ.get("GCP_PROJECT")
+            return client, project_id
+        except Exception as e:
+            raise HTTPException(
+                status_code=500,
+                detail=f"Failed to initialize Secret Manager client: {str(e)}"
+            )
+
+    @staticmethod
+    def get_secret_id(provider: Literal["openai", "anthropic"], customer_id: str):
+        if os.getenv("isDevelopmentMode") == "enabled":
```

Ruff (0.8.2)

Use capitalized environment variable `ISDEVELOPMENTMODE` instead of `isDevelopmentMode`

(SIM112)